generators/app/index.js

```diff
@@ -125,6 +151,18 @@ module.exports = class extends BaseGenerator {
         } catch (err) {
             this.log(`${chalk.red.bold('WARN!')} Could not register as a jhipster entity post creation hook...\n`);
         }
+
+        this.entityClass = this.props.entityClass;
+
+        this.template('src/main/java/service/GenericConsumer.java.ejs', `${javaDir}service/GenericConsumer.java`);
+
+        if (this.props.components.includes('consumer')) {
+            this.template('src/main/java/service/EntityConsumer.java.ejs', `${javaDir}service/${this.entityClass}Consumer.java`);
```

Comment:
It needs to be discussed as we have several option:
- keep `service` package
- move into `service/kafka`
- move into `service/producer` or `service/consumer`
- move into `service/kafka/producer` or `service/kafka/consumer`
etc

WDYT ?

Reply:
My opinion:

- keep `service` package

No, there will be too many classes here

- move into `service/kafka`

No, I think having 'producer' and 'consumer' concepts is a good 'candidate' to have dedicated packages for this.

- move into `service/producer` or `service/consumer`

Good but what about GenericConsumer and GenericProducer (that might eventually being renamed to AbstractConsumer/AbstractProducer).
I don't know yet how it will be implemented.
It means we will have FooConsumer, BarConsumer and GenericConsumer in the same package, but GenericConsumer might not be a "real consumer"

- move into `service/kafka/producer`or `service/kafka/consumer`

Good, 
We can put FooConsumer, BarConsumer in service/kafka/consumer.
And GenericConsumer and GenericProducer (or AbstractConsumer/AbtractProducer) in `service/kafka`

It might be too early to define this as we don't know exactly what kind of classes we will need..

Reply:
I agree with @ctamisier, the last option seems to be the best but maybe it's too early to fix it?